From: Rok Mandeljc <rok.mandeljc@gmail.com>
Date: Mon, 14 Apr 2014 15:02:37 +0200
Subject: [PATCH 1/1] Removed HAVE_STDINT guards in flip1.c and flip2.c since
 they cause errors under MinGW

--- a/flip1.c
+++ b/flip1.c
@@ -29,12 +29,7 @@
 #include <errno.h>
 #include <limits.h>
 #include <unistd.h>
-
-#if HAVE_STDINT_H
 #include <stdint.h>
-#elif HAVE_INTTYPES_H
-#include <inttypes.h>
-#endif
 
 #include "flip1.h"
 #include "dfu.h"

--- a/flip2.c
+++ b/flip2.c
@@ -26,12 +26,7 @@
 #include <errno.h>
 #include <limits.h>
 #include <unistd.h>
-
-#if HAVE_STDINT_H
 #include <stdint.h>
-#elif HAVE_INTTYPES_H
-#include <inttypes.h>
-#endif
 
 #include "flip2.h"
 #include "dfu.h"
-- 
1.9.0

