--- a/configure.in
+++ b/configure.in
@@ -263,7 +263,7 @@
         kernel=`uname -r`
 
         case "${kernel}" in
-             2.6.*|3.*) 
+             2.6.*|3.*|4.*) 
         	    AC_MSG_RESULT([2.6.x/3.x family (${kernel})])
         	    AC_DEFINE([KERNEL_2_6], 1, [Define to 1 if you're using Linux 2.6.x or Linux 3.x])
         	    ;;
@@ -272,14 +272,14 @@
         	    AC_MSG_RESULT([2.4 family (${kernel})])
         	    AC_DEFINE([KERNEL_2_4], 1, [Define to 1 if you're using Linux 2.4.x])
         	    ;;
-             3.*)
+             3.*|4.*)
                     AC_MSG_RESULT([3 family (${kernel})])
                     AC_DEFINE([KERNEL_3], 1, [Define to 1 if you're using Linux 3.x])
                     ;;
 
              *) 
         	    AC_MSG_RESULT([unsupported (${kernel})])
-        	    AC_MSG_ERROR([Valgrind works on kernels 2.4, 2.6, 3.x])
+        	    AC_MSG_ERROR([Valgrind works on kernels 2.4, 2.6, 3.x, 4.x])
         	    ;;
         esac
 
