From eb5ad2a0ab6605a0682888b30251697e36bc02bf Mon Sep 17 00:00:00 2001
From: "Build @Spectrum" <shiqi.f@pervices.com>
Date: Fri, 13 Dec 2019 21:13:22 +0000
Subject: [PATCH 1/2] Enable nand command by default

---
 include/configs/socfpga_common.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/socfpga_common.h b/include/configs/socfpga_common.h
index d8dbe6e..4a92a4c 100644
--- a/include/configs/socfpga_common.h
+++ b/include/configs/socfpga_common.h
@@ -512,7 +512,7 @@
 
 /* NAND */
 #if (CONFIG_PRELOADER_BOOT_FROM_NAND == 0)
-#undef CONFIG_NAND_DENALI
+#define CONFIG_NAND_DENALI
 #else
 #define CONFIG_NAND_DENALI
 #endif
-- 
2.7.4

