From c150f479f9aca50e4eec64cd2c6aecb0b0a13ff3 Mon Sep 17 00:00:00 2001
From: Shiqi Feng <shiqi.f@pervices.com>
Date: Mon, 10 Apr 2017 22:52:20 +0000
Subject: [PATCH] U-BOOT: Boot from NAND flash

---
 board/altera/socfpga/build.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/board/altera/socfpga/build.h b/board/altera/socfpga/build.h
index c1f10c0..ebacf39 100644
--- a/board/altera/socfpga/build.h
+++ b/board/altera/socfpga/build.h
@@ -36,8 +36,8 @@
  * Only 1 boot option to be enabled at any time
  */
 #define CONFIG_PRELOADER_BOOT_FROM_QSPI		(0)
-#define CONFIG_PRELOADER_BOOT_FROM_SDMMC	(1)
-#define CONFIG_PRELOADER_BOOT_FROM_NAND		(0)
+#define CONFIG_PRELOADER_BOOT_FROM_SDMMC	(0)
+#define CONFIG_PRELOADER_BOOT_FROM_NAND		(1)
 #define CONFIG_PRELOADER_BOOT_FROM_RAM		(0)
 
 /*
-- 
2.7.4

