From b14e219178d0fddc2526ff1c799b0de3eec9b282 Mon Sep 17 00:00:00 2001
From: "Build @Spectrum" <tech@pervices.com>
Date: Thu, 18 May 2017 19:54:22 +0000
Subject: [PATCH] Enable nand command by default

---
 include/configs/socfpga_common.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/socfpga_common.h b/include/configs/socfpga_common.h
index 6d1424a..5c22242 100644
--- a/include/configs/socfpga_common.h
+++ b/include/configs/socfpga_common.h
@@ -508,7 +508,7 @@
 
 /* NAND */
 #if (CONFIG_PRELOADER_BOOT_FROM_NAND == 0)
-#undef CONFIG_NAND_DENALI
+#define CONFIG_NAND_DENALI
 #else
 #define CONFIG_NAND_DENALI
 #endif
-- 
2.7.4

