From a4113a3c666e2598a9d4c248c9179c1584b0dc75 Mon Sep 17 00:00:00 2001
From: "Build @Spectrum" <shiqi.f@pervices.com>
Date: Fri, 13 Dec 2019 21:15:12 +0000
Subject: [PATCH 2/2] Enable sd card read only file system

---
 include/configs/socfpga_common.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/include/configs/socfpga_common.h b/include/configs/socfpga_common.h
index 4a92a4c..f57e13e 100644
--- a/include/configs/socfpga_common.h
+++ b/include/configs/socfpga_common.h
@@ -193,6 +193,7 @@
 	"mmcloadcmd=fatload\0" \
 	"mmcloadpart=1\0" \
 	"mmcroot=/dev/mmcblk0p2\0" \
+	"mmcrootfstype=squashfs\0" \
 	"qspiloadcs=0\0" \
 	"qspibootimageaddr=0xa0000\0" \
 	"qspifdtaddr=0x50000\0" \
@@ -208,7 +209,7 @@
 		"${mmcloadcmd} mmc 0:${mmcloadpart} ${loadaddr} ${bootimage};" \
 		"${mmcloadcmd} mmc 0:${mmcloadpart} ${fdtaddr} ${fdtimage}\0" \
 	"mmcboot=setenv bootargs " CONFIG_BOOTARGS \
-		" root=${mmcroot} rw rootwait;" \
+		" root=${mmcroot} rw rootfstype=${mmcrootfstype} rootwait;" \
 		"bootz ${loadaddr} - ${fdtaddr}\0" \
 	"netboot=dhcp ${bootimage} ; " \
 		"tftp ${fdtaddr} ${fdtimage} ; run ramboot\0" \
-- 
2.7.4

