From 2d98111cfc28cca0447a43326dfb3a17c9611ebe Mon Sep 17 00:00:00 2001
From: Victoria S <victoria.s@pervices.com>
Date: Thu, 17 Nov 2022 18:01:55 +0000
Subject: [PATCH] Poky Patch for autotate

---
 autotate.sh | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/autotate.sh b/autotate.sh
index bd3bb36..453ad2c 100755
--- a/autotate.sh
+++ b/autotate.sh
@@ -75,14 +75,12 @@ elif [ $# -eq 5 ]; then
 
     ./configure                         \
             --prefix=/usr                   \
-            --host=x86_64                   \
-            CC=aarch64-linux-gnu-gcc        \
+            --host=arm-unknown-linux-gnueabihf                   \
             CFLAGS="-Wall -O3 -pipe -fomit-frame-pointer -Wall \
                     -march=armv8-a -mtune=cortex-a53 \
                     -Werror -lm" \
             CPPFLAGS="-Wall -O3 -pipe -fomit-frame-pointer -Wall \
                     -march=armv8-a -mtune=cortex-a53 " \
-            CXX=aarch64-linux-gnu-g++       \
             CXXFLAGS="-Wall -O3 -pipe -fomit-frame-pointer -Wall \
                     -march=armv8-a -mtune=cortex-a53 " \
             PRODUCT=$1 \
-- 
2.34.0

