From ab49bebadb983927f781c676948a47959af3c410 Mon Sep 17 00:00:00 2001
From: Amir Ghanbari Bavarsad <amir.b@pervices.com>
Date: Thu, 28 Feb 2019 21:29:53 +0000
Subject: [PATCH 3/8] Bugfix, new chip revisions have a mask of 0xF8

---
 drivers/tty/serial/max310x.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/tty/serial/max310x.c b/drivers/tty/serial/max310x.c
index 8a3e92638e10..9dfedbe6c071 100644
--- a/drivers/tty/serial/max310x.c
+++ b/drivers/tty/serial/max310x.c
@@ -236,7 +236,7 @@
 
 /* Misc definitions */
 #define MAX310X_FIFO_SIZE		(128)
-#define MAX310x_REV_MASK		(0xfc)
+#define MAX310x_REV_MASK		(0xf8)
 
 /* MAX3107 specific */
 #define MAX3107_REV_ID			(0xa0)
-- 
2.20.1

